### ISSUE TITLE
Tables4BackEnd

### ISSUE THREAD

**Issue by Tfmenard** [RedpathMuseum/open-museum#42]
# Tours
+ int id
+ string name

# Exhibits
+ int id
+ string Name;
+ int tour_id
+ text description
  
# Annotations
+ int id
+ string name
+ string text (Text of Annotation)
+ int exihibit_id
+ double [3] camera_position;
+ double [3] camera_target;
+ double [3] marker_position;

# Relations
+ TourTable has One-to-Many relation with ExhibitTable
+ ExhibitTable has One-to-Many relation with AnnotationSetTable
+ AnnotationSetTable has One-to-Many relation with AnnoationObjTable

**Comment by Tfmenard** [comment]
@fxleblanc  C pas si pire Markdown finalement lol. Est-ce que t'as besoin de plus détails ?

**Comment by fxleblanc** [comment]
@Tfmenard ça me semble bon. Voici ma proposition pour les noms des tables:
- tours
- exhibits

Pour AnnotationObj et AnnotationSet, est-il possible de ramener tout ça à une table annotations?

**Comment by Tfmenard** [comment]
@fxleblanc Pour AnnotationObj et AnnotationSet, oui c'est possible, il faut seulement que le data soit sauvegardé en queue. Je viens de corriger le comment

**Comment by fxleblanc** [comment]
@Tfmenard , j'ai fait quelques ajustements à ta conception, tu me diras ce que tu en penses

**Comment by Tfmenard** [comment]
@fxleblanc  OUais c'est bon comme ca, ce sera plus efficace et ça structurera bien les URLs. J'imagine c'était ça l'idée ?

**Comment by fxleblanc** [comment]
Oui. La structure des urls est séparée de la structure de la base de
données.
